docs/quickstart.rst

```diff
@@ -1,11 +1,11 @@
-Nexmo Client Library for Python
+Vonage Client Library for Python
 ===============================
 
 |PyPI version| |Build Status|
 
-This is the Python client library for Nexmo's API. To use it you'll need
-a Nexmo account. Sign up `for free at
-nexmo.com <https://dashboard.nexmo.com/sign-up?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__.
+This is the Python client library for Vonage's API. To use it you'll need
+a Vonage account. Sign up `for free at
+vonage.com <https://dashboard.vonage.com/sign-up?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__.
```

Comment:
The dasboard url didn't change. The link should still be dashboard.nexmo.com at the moment